### DISCUSSION TITLE
Advice needed about incorporating read depth

### DISCUSSION THREAD

**Discussion by RachBioHaz** [FrederickHuangLin/ANCOMBC#159] [General]
Hi, I have a dataset that has some inherent structural differences in the regions sampled that affect expected read depth. There is a correlation between the read depth with the region sampled. I am looking at the data in two ways: on a region-by-region basis and as a landscape as a whole. 
My question is: What is the best way of adding read depth to the ANCOM-BC model for both approaches? I am fairly new to community analysis and R, so would appreciate some advice on the approach as well as the syntax for this package.

Cheers

**Comment by FrederickHuangLin** [comment]
Hi @RachBioHaz,

Generally, you do not have to worry about sequencing depth using ANCOM-BC since its methodology was developed to correct this bias. 

If the sequencing depths are vastly different, you can put it in the `formula` argument, for instance, `formula = "depth + group"`; however, based on our simulations and real data applications, the results remained similar most of the time.

Best,
Huang

**Comment by RachBioHaz** [comment]
Thank you. I hoped I had understood the documentation correctly.CheersRacheleSent from my Galaxy
-------- Original message --------From: Frederick Huang Lin ***@***.***> Date: 30/3/23  07:12  (GMT+10:00) To: FrederickHuangLin/ANCOMBC ***@***.***> Cc: Rachele Gore ***@***.***>, Mention ***@***.***> Subject: Re: [FrederickHuangLin/ANCOMBC] Advice needed about incorporating
  read depth (Discussion #159) 
Hi @RachBioHaz,
Generally, you do not have to worry about sequencing depth using ANCOM-BC since its methodology was developed to correct this bias.
If the sequencing depths are vastly different, you can put it in the formula argument, for instance, formula = depth + group; however, based on our simulations and real data applications, the results remained similar most of the time.
Best,
Huang
